packages/fast-components-react-base/src/auto-suggest/index.ts

```diff
@@ -1,4 +1,6 @@
 import AutoSuggest from "./auto-suggest";
+import autoSuggestSchema from "./auto-suggest.schema";
 
+export { autoSuggestSchema };
```

Comment:
I think we should keep naming intuitive here and elsewhere: `AutoSuggestSchema`

Reply:
Chatted offline